### DISCUSSION TITLE
Using the Blocks are effectively

### DISCUSSION THREAD

**Discussion by onurhan1337** [shadcn-ui/ui#3325] [Q&A]
Block is really a cool innovation. However, when I started to use it in one of my projects, I felt that there was a missing point. I would like to share my views, believing that we can brainstorm what can be done about it.

My main problem is the monolithic file: when I want to get only one related component in Blocks (sidebar for example) it is really complicated. It's hard to find the component I want, at least the components could be separated with comment lines. 

What do you think about this, is there a method you use to quickly implement it in your project?

**Comment by refacta** [comment]
_Lift Mode_ was implemented recently to solve this. Simply toggle on 'Lift Mode' above each block, then hover over the component you need to extract. Then when you mouseover a component, a clipboard icon will appear top right of that component, which will allow you to copy the code pertaining to that component only.
